composer/trainer/trainer.py

```diff
@@ -1159,6 +1160,9 @@ def __init__(
             self.local_hparams = extract_hparams(locals())
             self.logger.log_hyperparameters(self.local_hparams)
 
+        # Log composer version.
+        self.logger.log_hyperparameters({'composer_version': composer_version})
```

Comment:
if we're going to log this, lets also log the git commit if it exists? there is a utility func for this already in `collect_env.py`

Reply:
fixd